translations/translations.json

```diff
@@ -6667,6 +6667,8 @@
       "detailsInformation": "Informationen",
       "detailsNextPoi": "nächster",
       "detailsPreviousPoi": "vorheriger",
+      "previousPoi": "Vorheriger Ort",
```

Comment:
No need to add these new translations you can just use ${`detailsPreviousPoi` `poi`}.

Reply:
Not the case for every language, in some languages this (i.e. the order) could differ.